src/js/components/Button/stories/IconLabel.js

```diff
@@ -4,16 +4,35 @@ import { Add } from 'grommet-icons';
 
 import { Box, Button, Grommet } from 'grommet';
 import { grommet } from 'grommet/themes';
+import { deepMerge } from '../../../utils';
+
+const customButtonIconColor = deepMerge(grommet, {
+  button: {
+    color: { light: 'brand', dark: 'accent-2' },
+  },
+});
 
 const IconLabel = () => (
-  <Grommet theme={grommet}>
-    <Box align="center" pad="large">
+  <Grommet theme={customButtonIconColor}>
+    <Box background="dark-2" align="center" pad="large">
       <Box round="full" overflow="hidden" background="neutral-1">
         <Button icon={<Add />} hoverIndicator onClick={() => {}} />
       </Box>
       <Box align="center" pad="large" gap="small">
-        <Button icon={<Add />} label="Add" onClick={() => {}} primary />
-        <Button icon={<Add />} label="Add" onClick={() => {}} />
+        <Button
+          color="black"
+          icon={<Add />}
+          label="Add"
+          onClick={() => {}}
+          primary
+        />
+        <Button
+          color="white"
+          icon={<Add />}
+          label="Add"
+          onClick={() => {}}
+          primary
+        />
```

Comment:
In these examples, the color of the text doesn't match the color of the icon like the PR suggests. am I missing something?

Reply:
I was cleaning up merge conflicts I believe I overridden my logic changes with the masters original. I just posted them back and will take a look at the testing changes as well as the chromatic changes you had pointed out